process/peer/process.go

```diff
@@ -364,7 +364,13 @@ func (vs *validatorStatistics) UpdatePeerState(header data.MetaHeaderHandler, ca
 	if err != nil {
 		return nil, err
 	}
-	leaderPK := core.GetTrimmedPk(vs.pubkeyConv.Encode(consensusGroup[0].PubKey()))
+
+	encodedLeaderPk, err := vs.pubkeyConv.Encode(consensusGroup[0].PubKey())
```

Comment:
we definitely need an `import-db` to guarantee that there is nothing breaking

Reply:
in this case, use silent, since the value is only used in logging

Reply:
changed